test/minters/redeem/ZoraCreatorRedeemMinterFactory.t.sol

```diff
@@ -16,64 +16,65 @@ import {IZoraCreator1155Factory} from "../../../src/interfaces/IZoraCreator1155F
 import {ZoraCreatorRedeemMinterStrategy} from "../../../src/minters/redeem/ZoraCreatorRedeemMinterStrategy.sol";
 import {ZoraCreatorRedeemMinterFactory} from "../../../src/minters/redeem/ZoraCreatorRedeemMinterFactory.sol";
 
-contract ZoraCreatorRedeemMinterFactoryTest is Test {
-    ProtocolRewards internal protocolRewards;
-    ZoraCreator1155Impl internal target;
-    ZoraCreatorRedeemMinterFactory internal minterFactory;
-    address payable internal tokenAdmin = payable(address(0x999));
-    address payable internal factoryAdmin = payable(address(0x888));
-    address internal zora;
+/// @notice v1.4.1 will not support burn to redeem
```

Comment:
Should we confirm with the product team this is the case? We can update / redeploy the contracts as needed with the fix.

Reply:
yep will do